recipes/fast-dds/all/conanfile.py

```diff
@@ -49,18 +49,13 @@ def _min_cppstd(self):
 
     @property
     def _compilers_minimum_version(self):
-        if Version(self.version) < "2.11.0":
-            return {
-                "gcc": "8",
-                "clang": "12",
-                "apple-clang": "12",
-            }
-        else:
-            return {
-                "gcc": "9",
-                "clang": "15",
-                "apple-clang": "15",
-            }
+        return {
+            "gcc": "8",
+            "clang": "12",
+            "apple-clang": "12",
```

Comment:
These are 15 in the newer version, is it true that it compiles with older ones too?

Reply:
And this is building latest main - so that is the latest version of fast-dds